CMake/Modules/Flow123d_config_header_create.cmake

```diff
@@ -0,0 +1,10 @@
+### CMake script for creating config header file
+MESSAGE (STATUS "CREATING CONFIG.H")
+
+# get tmp file path from given FILE_PATH variable (-DFILE_PATH=path/to/file.extension)
+set(TMP_FILE_PATH "${FILE_PATH}.tmp")
```

Comment:
Use more descriptive name then FILE_PATH and describe this variable as parameter of the script.